src/Sql/Sql/LedgerDigestUploads/Cmdlet/EnableAzSqlDatabaseLedgerDigestUpload.cs

```diff
@@ -0,0 +1,105 @@
+﻿// ----------------------------------------------------------------------------------
+//
+// Copyright Microsoft Corporation
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+// http://www.apache.org/licenses/LICENSE-2.0
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+// ----------------------------------------------------------------------------------
+
+using System;
+using System.Linq;
+using System.Collections.Generic;
+using System.Management.Automation;
+using Microsoft.Azure.Commands.ResourceManager.Common.ArgumentCompleters;
+using Microsoft.Azure.Commands.Sql.LedgerDigestUploads.Model;
+using Microsoft.Azure.Commands.Sql.Database.Model;
+using Microsoft.Azure.Management.Internal.Resources.Utilities.Models;
+namespace Microsoft.Azure.Commands.Sql.LedgerDigestUploads.Cmdlet
+{
+    /// <summary>
+    /// Defines the Enable-AzSqlDatabaseLedgerDigestUpload cmdlet
+    /// </summary>
+    [Cmdlet("Enable", ResourceManager.Common.AzureRMConstants.AzureRMPrefix + "SqlDatabaseLedgerDigestUpload", ConfirmImpact = ConfirmImpact.Low, SupportsShouldProcess = true), OutputType(typeof(AzureSqlDatabaseLedgerDigestLocationModel))]
+    public class EnableAzSqlDatabaseLedgerDigestUpload : AzureSqlDatabaseLedgerDigestUploadBase
+    {
+        /// <summary>
+        /// Gets or sets the Azure Sql Database Ledger Digest Uploads endpoint
+        /// </summary>
+        [Parameter(Mandatory = true,
+            ValueFromPipelineByPropertyName = true,
+            Position = 3,
+            HelpMessage = "The Azure Sql Database Ledger Digest Uploads endpoint.")]
+        [ValidateNotNullOrEmpty]
+        public string Endpoint { get; set; }
+
+        /// <summary>
```

Comment:
Isn't this logic commong for enable and disable? Should it go to the base class?

Reply:
endpoint is not used in disable